product_multi_category/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas, Guewen Baconnier, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product Multi Categories",
+    "version": "9.0.1.0.0",
+    "author": "Sharoon Thomas,Sodexis,Odoo Community Association (OCA)",
```

Comment:
You should put here the rest of the contributors: Akretion, Camptocamp and Avanzosc